app/src/main/java/cm/aptoide/pt/home/apps/AppsManager.java

```diff
@@ -82,6 +84,7 @@ public AppsManager(UpdatesManager updatesManager, InstallManager installManager,
 
   public Observable<List<App>> getDownloadApps() {
     return installManager.getInstallations()
+        .throttleFirst(200, TimeUnit.MILLISECONDS)
```

Comment:
This throttleFirst method will ignore some emissions. If he is on the "waiting" stage it will ignore the emission. If that emission is the Download Completed one, then it will ignore it.